src/ios/CDVAdMob.m

```diff
@@ -719,6 +751,19 @@ - (void)resizeViews {
                         bf.origin.y -= parentView.safeAreaInsets.bottom;
                         bf.size.width = wf.size.width - parentView.safeAreaInsets.left - parentView.safeAreaInsets.right;
                         wf.size.height -= parentView.safeAreaInsets.bottom;
+
+                        //If safeAreBackground was turned turned off, turn it back on
+                        if( _safeAreaBackgroundView.hidden ){
+        					_safeAreaBackgroundView.hidden = false;
```

Comment:
You can remove the if statement here. Instead of
``` ObjC
if( _safeAreaBackgroundView.hidden ){
 	_safeAreaBackgroundView.hidden = false;
}
```
you can just write:
``` ObjC
_safeAreaBackgroundView.hidden = false;
```

Reply:
Great suggestion for simplification I will make the change.